buildSrc/src/main/kotlin/Ci.kt

```diff
@@ -1,19 +1,24 @@
 object Ci {
 
-   // this is the version used for building snapshots
-   // .buildnumber-snapshot will be appended
-   private const val snapshotBase = "6.0.0"
+   /**
+    * The base version used for the release version.
+    *
+    * `-SNAPSHOT` or `-LOCAL` will be appended.
+    */
+   private const val snapshotBase = "5.10.0"
```

Comment:
```suggestion
   private const val snapshotBase = "6.0.0"
```

Or is there a reason for keeping the old version?

Reply:
Oops, that's a mistake. Thanks for checking. I made this PR before the version was updated to 6.0.0.